detection_rules/packaging.py

```diff
@@ -256,6 +264,9 @@ def save(self, verbose=True):
         self._package_kibana_index_file(rules_dir)
 
         if self.release:
+            if self.registry_data:
```

Comment:
i would be explicit what you're checking against here
```suggestion
            if self.registry_data is not None:
```

Reply:
actually just going to remove it - essentially any packaged marked for release will require `registry` data and if not, it will fail `assertion` or validations in `_generate_registry_package`